packages/angular/build/src/builders/application/schema.json

```diff
@@ -37,6 +37,33 @@
       "type": "string",
       "description": "Customize the base path for the URLs of resources in 'index.html' and component stylesheets. This option is only necessary for specific deployment scenarios, such as with Angular Elements or when utilizing different CDN locations."
     },
+    "security": {
```

Comment:
Is `security` too broad a grouping? I'm not entirely sure what else we'd put in here. I'm wondering if we should drop this object and just have `autoCsp` as a top-level option? WDYT @alan-agius4, @clydin?

Reply:
For now, I think we only have `autoCsp` to add to this top-level object, but this is a logical location for grouping other secure-by-default options that we may pursue in the future. If you feel strongly about not having this nesting for this release, I suppose we can go to a top-level option. Also always happy to chat about what other secure-by-default features we can add beyond auto-CSP!

Reply:
We can keep `security` just to leave the door open for future features in this space. Always happy to discuss making Angular more secure-by-default!

Reply:
Excited to work further with y'all's! Thanks!